Makefile

```diff
@@ -7,10 +7,12 @@ export GO111MODULE=on
 .PHONY: upm
 upm: cmd/upm/upm ## Build the UPM binary
 
-cmd/upm/upm: $(SOURCES) $(RESOURCES) $(GENERATED)
-	go run github.com/rakyll/statik -src resources -dest internal -f
+cmd/upm/upm: $(SOURCES) $(RESOURCES) $(GENERATED) statik
 	cd cmd/upm && go build -ldflags "-X 'github.com/replit/upm/internal/cli.version=$${VERSION:-development version}'"
 
+statik:
```

Comment:
small nit: Make depends on the left-hand-side of every rule be a path that is generated by the rule. if it does not create it, this rule will always be "out of date", and successive `make` invocations will always try to run this:

```shell
lhchavez@luisval:~/upm$ make cmd/upm/upm
go run github.com/rakyll/statik -src resources -dest internal -f
cd cmd/upm && go build -ldflags "-X 'github.com/replit/upm/internal/cli.version=${VERSION:-development version}'"
lhchavez@luisval:~/upm$ make cmd/upm/upm
go run github.com/rakyll/statik -src resources -dest internal -f
cd cmd/upm && go build -ldflags "-X 'github.com/replit/upm/internal/cli.version=${VERSION:-development version}'"
lhchavez@luisval:~/upm$ make cmd/upm/upm
go run github.com/rakyll/statik -src resources -dest internal -f
cd cmd/upm && go build -ldflags "-X 'github.com/replit/upm/internal/cli.version=${VERSION:-development version}'"
```

_so_ the correct way to express this is the following:

```suggestion
internal/statik/statik.go: $(shell find resources -type f)
```

and have L10 and L72 depend on `internal/statik/statik.go` instead of `statik`. the `$(shell ...)` expression also has the effect of forcing this rule to be marked as stale if any of the files in `resources/` change.

Reply:
:+1: Will create a follow up PR